first-task/coverage/LengthConverte.js.html

```diff
@@ -0,0 +1,366 @@
+<!doctype html>
```

Comment:
Delete all the generated stuff by `coverage` from PR. Nothing generated should come to repository by default.

Simply add `coverage` dir to gitignore.

Avoid staging of everything in commit, think carefully what you need to commit.